client/config.go

```diff
@@ -110,15 +108,15 @@ func (c *Config) ValidateAndSetDefaults() error {
 	}
 
     // Only validate or set defaults for Icmp if it's not nil
-    if c.Icmp != nil {
+    if c.ICMPConfig != nil {
         // limit for pro-ping, below 25 it's not working
-        if c.Icmp.Size < 25 {
-            c.Icmp.Size = 25
+        if c.ICMPConfig.Size < 25 {
+            c.ICMPConfig.Size = 25
         }
     } else {
         // If Icmp is nil, let's initialize it with default values
-        c.Icmp = &Icmp{
-            Df:   false,
+        c.ICMPConfig = &ICMPConfig{
+            DF:   false,
             Size: 25,  
```

Comment:
Based on what I just looked at, the current default value is 25 (on my machine, at least 🤣)

![image](https://github.com/TwiN/gatus/assets/15699766/35cc1c65-45e6-4765-ba1a-868674d15b67)

Side note, I also see that there's a comment on L112 that may no longer be valid - it says `// limit for pro-ping, below 25 it's not working`.

Reply:
I've set this configuration because it's the only way I've found to make it work.
But maybe I'm missing something in Go.

I did the tests by validating with network captures:
- A classic linux ping can have a data of 0.
- If I don't specify a Size or defaultSize by default the Size is 24.
*So you are right I can remove defaultSize = 24*

But then, If I don't specifiy anything in icmp in my yaml file, the ping is not working.

That's why I put this code with the Size in it.
```
        // If Icmp is nil, let's initialize it with default values
        c.ICMPConfig = &ICMPConfig{
            DF:   false,
            Size: 24,
        }
```
But maybe this is not the good way to initialise it.

If I test with a Size <24 it doesn't work either.
I notice that this limitation exists in other tools that use pro-bing
For example
https://github.com/SuperQ/smokeping_prober
`Packetdata size in bytes. Default 56 (Range: 24 - 65535) ` 

That's why I put
```
    if c.ICMPConfig != nil {
        // limit for pro-ping, below 24 it's not working
        if c.ICMPConfig.Size < 24 {
            c.ICMPConfig.Size = 24
        }
```

And my mistake: the limit is 24, not 25.